### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by KseniaKuntsevich**
Suppose to work fine after merge (without body container)
Add footer with color, container.
Lint checked

### CODE REVIEW

File: src/components/Footer/FooterBottom/index.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.bg-deep-gray {
+	background-color: #41414E;
+	color: black;
```
Comment (Xelsoid) [inline src/components/Footer/FooterBottom/index.css:3]: image
I hope you can choose smth from this color palette

File: src/components/Footer/FooterNav/index.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.footerLink:hover {
+	color: #C4A871;
```
Comment (Xelsoid) [inline src/components/Footer/FooterNav/index.css:2]: check color please

File: src/components/Footer/index.css [hunk 3]
```diff
@@ -1,11 +1,6 @@
...
-  position: relative;
-  text-align: center;
-  border-top: 1px solid #ccc;
-  padding-top: 10px;
-  margin: 20px auto 30px;
+.bg-gray {
+	background-color: #5B5B65;
+}
+.text-black {
+	color: black
```
Comment (Xelsoid) [inline src/components/Footer/index.css:5]: the same

File: src/components/Footer/FooterBottom/index.css (hunk 1 above)
Comment (KseniaKuntsevich) [reply src/components/Footer/FooterBottom/index.css:3]: My background precisely #41414Е and text suppose to be black inside footer)

**Comment by maryna-js** [comment]
looks good

**Comment by Xelsoid** [comment]
fine

**Pull request merged**